source/widgets/base_build_widget.py

```diff
@@ -38,6 +38,9 @@ def show_release_notes(self):
             v = re.sub(r"\D", "", str(self.build_info.semversion.finalize_version()))
 
             webbrowser.open(f"https://www.blender.org/download/lts/#lts-release-{v}")
+        elif self.build_info.branch == "bforartists":
```

Comment:
How is this going? I think the easiest route would just be to link here: https://www.bforartists.de/release-notes/

Reply:
yep working on it, I just wanted it to link to the correct per version release notes directly, but indeed I agree with you, if I wasn't able to get direct link to per version release notes I'll use the link, sadly the bforartists de release notes link don't have correct HTML element tag IDs so we can have per version URLs so my current thinking is NextCloud release notes text files